### PR TITLE
MBS-8005 fix typos and supplement documentation

### PR DESCRIPTION
**Pull request by int02h**

### CODE REVIEW

**Review by eugene-krivobokov — APPROVED** [review]

File: docs/content/docs/ide/Benchmarking.md [hunk 1]
```diff
@@ -5,20 +5,21 @@ type: docs
 
 # Бенчмарки IDE
```
Comment (eugene-krivobokov) [inline docs/content/docs/ide/Benchmarking.md:7]: Let's add "avito" disclaimer. 
We have a dedicated shortcode. https://avito-tech.github.io/avito-android/docs/contributing/docs/#internal-links

Comment (int02h) [reply docs/content/docs/ide/Benchmarking.md:7]: I've added more info about plugin's disfunction

File: docs/content/docs/ide/Benchmarking.md [hunk 2]
```diff
@@ -5,20 +5,35 @@ type: docs
...
 
-Используем [Performance testing plugin](https://intellij-support.jetbrains.com/hc/en-us/articles/207241225-Performance-testing-plugin-in-PhpStorm).
+{{< hint info>}} Здесь содержиться информация, специфичная для Авито {{< /hint >}}
```
Comment (eugene-krivobokov) [inline docs/content/docs/ide/Benchmarking.md]: `{{< avito >}}` ?

**Review by eugene-krivobokov — APPROVED** [review]

File: docs/content/docs/ide/Benchmarking.md (hunk 2 above)
Comment (int02h) [reply docs/content/docs/ide/Benchmarking.md]: It adds an English disclaimer but the other parts of document are in Russian. Btw I've changed the doc as you suggested.

**Pull request merged**